### DISCUSSION TITLE
DNS over TLS seems to fail?

### DISCUSSION THREAD

**Discussion by captainkev76** [qdm12/gluetun#1572] [Q&A] [answered]
Hi All,

I'm a first-time Gluetun user, and followed the Wiki instructions to setup Wireguard for Surfshark, but I seem to be getting continuous problems with DNS-over-TLS.

The spoecific message in the logs is: [dns over tls] cannot update files: Get "https://www.internic.net/domain/named.root": context deadline exceeded (Client.Timeout exceeded while awaiting headers). I've attached the full logs, as well as a redacted docker-compose file.

Anyone come across this before?
[_gluetun_logs.txt](https://github.com/qdm12/gluetun/files/11501462/_gluetun_logs.txt)
[docker-compose.txt](https://github.com/qdm12/gluetun/files/11501464/docker-compose.txt)

**Comment by frepke** [comment, marked as answer]
Your should change the WIREGUEARD_ADDRESS to `- WIREGUARD_ADDRESSES=10.14.0.2/16`
You can check the ip-address in the .conf file downloaded from Surfshark: [https://my.surfshark.com/vpn/manual-setup/main/wireguard/own-key](https://my.surfshark.com/vpn/manual-setup/main/wireguard/own-key) Tab `Locations`

**Reply by captainkev76** [reply]
Thank you sir - that was the silver bullet! You're a star.

**Reply by frepke** [reply]
Graag gedaan
